functions/__sf_section_kubecontext.fish

```diff
@@ -0,0 +1,32 @@
+#
+#  Kubernetes (kubectl)
+#
+# Kubernetes is an open-source system for deployment, scaling,
+# and management of containerized applications.
+# Link: https://kubernetes.io/
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+function __sf_section_kubecontext -d "Display the kubernetes context"
+	__sf_util_set_default SPACEFISH_KUBECONTEXT_SHOW true
+	__sf_util_set_default SPACEFISH_KUBECONTEXT_PREFIX " "
+  __sf_util_set_default SPACEFISH_KUBECONTEXT_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+	__sf_util_set_default SPACEFISH_KUBECONTEXT_SYMBOL "☸️"
+	__sf_util_set_default SPACEFISH_KUBECONTEXT_COLOR cyan
+
+	[ $SPACEFISH_KUBECONTEXT_SHOW = false ]; and return
+
+	if not type -q kubectl
+		return
+	end
+
+	set -l sf_kube_context (kubectl config current-context 2>/dev/null)
+
+	__sf_lib_section \
+		$SPACEFISH_KUBECONTEXT_COLOR \
+		$SPACEFISH_KUBECONTEXT_PREFIX \
+		"$SPACEFISH_KUBECONTEXT_SYMBOL $sf_kube_context" \
```

Comment:
If we're going to add the space directly to the symbol (mentioned in the above comment), I would suggest then removing the space here.